crates/binstalk/src/fetchers/gh_crate_meta.rs

```diff
@@ -115,23 +118,42 @@ impl super::Fetcher for GhCrateMeta {
                     // just a best-effort
                     pkg_fmt = PkgFmt::guess_pkg_format(pkg_url);
 
+                    let crate_name = &self.data.name;
+                    let version = &self.data.version;
+                    let target = &self.target_data.target;
+
                     if pkg_fmt.is_none() {
                         return Err(InvalidPkgFmtError {
-                            crate_name: self.data.name.clone(),
-                            version: self.data.version.clone(),
-                            target: self.target_data.target.clone(),
+                            crate_name: crate_name.clone(),
+                            version: version.clone(),
+                            target: target.clone(),
                             pkg_url: pkg_url.to_string(),
                             reason: "pkg-fmt is not specified, yet pkg-url does not contain format, archive-format or archive-suffix which is required for automatically deducing pkg-fmt",
                         }
                         .into());
                     }
+
+                    warn!(
+                        concat!(
+                        "Crate {crate_name}@{version} on target {target} does not specify pkg-fmt ",
+                        "but its pkg-url also does not contain key format, archeve-format or ", 
```

Comment:
```suggestion
                        "but its pkg-url also does not contain key format, archive-format or ", 
```

Reply:
Also that block can just use line continuation instead of concat

Reply:
Done